### ISSUE TITLE
Docker container registry

### ISSUE THREAD

**Issue by DevSecNinja** [nowthis/sankeymatic#82]
Hi Steve,

This is a really cool site, thanks for making it! Are you planning to push a container to a container registry? You already have the Dockerfile so with this instruction that shouldn't be too difficult to push on every commit/release: https://docs.github.com/en/actions/use-cases-and-examples/publishing-packages/publishing-docker-images

I don't have access to the repo secrets, otherwise I would have proposed a PR. The Docker image allows others to easily deploy it in their home labs or on their web servers.

Thanks!

**Comment by tallguyjenks** [comment]
yes, please add it to a registry so i can get this spun up in my home lab!
